doc/version_history.rst

```diff
@@ -2,6 +2,12 @@
 Version History
 ===============
 
+v0.4.3
+------
+
+* Fix the bug to connect to wrong host.
```

Comment:
```
* Properly reports cause when connecting to non-existing host

Reply:
Fixed in the commit: [a96128e](https://github.com/lsst-ts/ts_m2com/pull/8/commits/a96128e332e3be4733dff30c0dedd2de2813cad8). Thanks!